API.Tests/Services/ParseScannedFilesTests.cs

```diff
@@ -349,4 +357,101 @@ public async Task ProcessFiles_ShouldCallFolderActionOnce()
 
 
     #endregion
+
+    [Fact]
+    public async Task HasSeriesFolderNotChangedSinceLastScan_AllSeriesFoldersHaveChanges()
+    {
+        const string testcase = "Subfolders always scanning all series changes - Manga.json";
+        var infos = new Dictionary<string, ComicInfo>();
+        var library = await _scannerHelper.GenerateScannerData(testcase, infos);
+
+        var testDirectoryPath = Path.Join(
+            Path.Join(Directory.GetCurrentDirectory(), "../../../Services/Test Data/ScannerService/ScanTests"),
+            testcase.Replace(".json", string.Empty));
+        testDirectoryPath = Path.GetFullPath(testDirectoryPath);
+        library.Folders = [new FolderPath() {Path = testDirectoryPath}];
+
+        _unitOfWork.LibraryRepository.Update(library);
+        await _unitOfWork.CommitAsync();
+
+        var fs = new FileSystem();
+        var ds = new DirectoryService(Substitute.For<ILogger<DirectoryService>>(), fs);
+        var scanner = _scannerHelper.CreateServices(ds, fs);
+        await scanner.ScanLibrary(library.Id);
+        var postLib = await _unitOfWork.LibraryRepository.GetLibraryForIdAsync(library.Id, LibraryIncludes.Series);
+
+        Assert.NotNull(postLib);
+        Assert.Equal(4, postLib.Series.Count);
+        var spiceAndWolf = postLib.Series.First(x => x.Name == "Spice and Wolf");
```

Comment:
This block of code is hard to parse through. Break up each fetch/assert by a new line.